template/src/dummy.scss

```
@@ -1,3 +1,5 @@
+@use 'src/assets/stylesheets/application';
```

```suggestion
@use 'src/assets/stylesheets/application' as app;
```

Open suggestion: Is it "lighter" if we add a shorter alias like `app`? 🤔 I'm imagining this would make scss codding slightly faster/easier, without removing the explicity of "application" (as app is commonly used). :)

---

Closing. I did not see the previous comment xD